### DISCUSSION TITLE
Gemma2 supported

### DISCUSSION THREAD

**Discussion by lordofthejars** [tjake/Jlama#65] [Ideas]
Hi mate, as you might know, I am researching security on prompting, jailbreak, prompt injection, ethic prompting, and so on.  And one approach is embedding these models into Java. I tried the ShieldGemma, which is a Gemma2 model. But it is not supported yet. Do you think it is hard to add support for Gemma2 models?

**Comment by tjake** [comment]
I’ll take a look it should work

**Reply by lordofthejars** [reply]
Great thanks.

The command I run was `jlama restapi google/shieldgemma-2b`

With the token

**Reply by lordofthejars** [reply]
Do you want I open as issue?

**Comment by tjake** [comment]
#84 added this, closing out this discussion
